packages/back-end/src/api/openapi/payload-schemas/PostMetricPayload.yaml

```diff
@@ -47,7 +47,17 @@ properties:
           - decrease
       cap:
         type: number
-        description: This should be non-negative
+        description: (deprecated, use capping and capValue fields instead) This should be non-negative
```

Comment:
Looks like according to the [OpenAPI docs](https://swagger.io/docs/specification/describing-parameters/) you can add `deprecated: true` and it should show up with proper styling for deprecated fields.